### PR TITLE
Add a demo kubeflow profile

### PR DESCRIPTION
**Pull request by 4n4nd**
since we disabled kubeflow registration in #219, I have added a demo profile with limited resources which can be accessed publicly

### CODE REVIEW

File: kubeflow-components/overlays/moc/profiles/demo-profile.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Profile
+metadata:
+    name: kf-demo
+spec:
+    owner:
+        kind: User
+        name: asanmukh@redhat.com
```
Comment (tumido) [inline kubeflow-components/overlays/moc/profiles/demo-profile.yaml:9]: we probably want to encrypt this field if it points to an user. Can this be set to a `Group` instead?

File: kubeflow-components/overlays/moc/profiles/demo-profile.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Profile
+metadata:
+    name: kf-demo
```
Comment (tumido) [inline kubeflow-components/overlays/moc/profiles/demo-profile.yaml:5]: This corresponds to the namespace created. Does it create the RoleBinding for the user?

**Review by tumido — CHANGES_REQUESTED** [review]
I think we need to have some basic documentation in the repo, before we start adding additional namespaces (not this one, but for other users)... Can you please include a quick README into this PR as well?


https://www.kubeflow.org/docs/components/multi-tenancy/getting-started/#managing-contributors-manually

File: kubeflow-components/overlays/moc/profiles/demo-profile.yaml (hunk 1 above)
Comment (4n4nd) [reply kubeflow-components/overlays/moc/profiles/demo-profile.yaml:9]: Good idea :+1:

File: kubeflow-components/overlays/moc/profiles/demo-profile.yaml (hunk 2 above)
Comment (4n4nd) [reply kubeflow-components/overlays/moc/profiles/demo-profile.yaml:5]: It does create a rolebinding for the user. It  gives a role called `kubeflow-admin` ([link](https://github.com/kubeflow/manifests/blob/f95406b14a2025db1e8b88d72caa1316ec5d1f49/kubeflow-roles/base/cluster-roles.yaml#L22))

**Comment by 4n4nd** [comment]
> before we start adding additional namespaces

Yeah we will need to figure out authentication before that, I have just added this namespace/profile so we can limit resource usage

**Review by tumido — COMMENTED** [review]
/lgtm Awesome stuff! :+1: :1st_place_medal:

**Comment by tumido** [comment]
it's still on hold, even though @4n4nd /unhold it. Let me try once more.

**Review by tumido — APPROVED** [review]

**Pull request merged**